src/main/java/dev/vasyl/proj/controller/BookController.java

```diff
@@ -33,4 +34,10 @@ public BookDto getBookById(@PathVariable Long id) {
     public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
         return bookService.save(requestDto);
     }
+
+    @PutMapping("/{id}")
+    public BookDto updateBookById(@PathVariable Long id,
+                                  @RequestBody CreateBookRequestDto requestDto) {
+        return bookService.update(id, requestDto);
+    }
```

Comment:
Check task description and add `DELETE /api/books/{id} (Delete a specific book)`

Reply:
In task description i don't have this api. It's task mistake?


"As a result of this HW you should have the following endpoints in the BookController:

Book Endpoints:

GET: /api/books (Retrieve book catalog) (should be done in the previous PR)
GET: /api/books/{id} (Retrieve book details) (should be done in the previous PR)
POST: /api/books (Create a new book) (should be done in the previous PR)
PUT: /api/books/{id} (Update a specific book)"